autoscheduler/scraper/management/commands/scrape_departments.py

```diff
@@ -0,0 +1,28 @@
+# Called to execute the beginning of scaping courses
+from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Department
+
+import time
+import datetime
+
+class Command(base.BaseCommand):
+    """ Gets all departments from banner and adds them to the database """
+
+    def scrape_departments():
+        term = add_arguments()x

```

Comment:
Are you running these changes before you commit them? This `x` should have been a quick find. Likewise, the `scrape_departments` function doesn't have a `term` parameter, and the `term = add_argument()` line shouldn't be there, as `add_argument()` isn't a function for us to call(it's for `Django` to use), nor does it return a string.